README.md

```diff
@@ -62,29 +62,57 @@ Goad will read your credentials from `~/.aws/credentials` or from the `AWS_ACCES
 
 ```sh
 # Get help:
-$ goad -h
-Usage of goad:
-  -c uint
-      number of concurrent requests (default 10)
-  -m string
-      HTTP method (default "GET")
-  -n uint
-      number of total requests to make (default 1000)
-  -r string
-      AWS regions to run in (comma separated, no spaces) (default "us-east-1")
-  -t uint
-      request timeout in seconds (default 15)
-  -u string
-      URL to load test (required)
-  -H string
-      HTTP Header to add to the request. This option can be set multiple times.
+$ goad --help
+usage: goad --url=URL [<flags>]
+
+An AWS Lambda powered load testing tool
+
+Flags:
+  -h, --help                   Show context-sensitive help (also try --help-long
+                               and --help-man).
+  -u, --url=URL                URL to load test
+  -m, --method="GET"           HTTP method
+  -b, --body=BODY              HTTP request body
+  -c, --concurrency=10         Number of concurrent requests
+  -n, --requests=1000          Total number of requests to make
+  -N, --timelimit=3600         Seconds to max. to spend on benchmarking
+  -t, --timeout=15             Request timeout in seconds
+  -r, --region=REGION ...      AWS regions to run in (repeat flag to run in more
```

Comment:
Okay, good point. I'll remove the short -r fag and keep only the long version to avoid conflict with AB.